piece_of_cake.py

```diff
@@ -0,0 +1,16 @@
+# get the prices of the ingredients ,the amount need and the optional - which is the ingredients we don't need
+# and calculate the price.
+def get_recipe_price(prices, optionals=[], **ingredients):
+    total_price = 0
+    for ingredient in ingredients:
+        ingredient_amount = int(ingredients[ingredient]) / 100
+        if ingredient in optionals:
+            continue
+        ingredient_price = ingredient_amount * prices[ingredient]
+        total_price = total_price + ingredient_price
+    print(total_price)
+
+
+get_recipe_price({'chocolate': 18, 'milk': 8}, chocolate=200, milk=100)
```

Comment:
include all of this in https://www.freecodecamp.org/news/if-name-main-python-example/
so this code will not be executed when importing this file